### PR TITLE
chore: update templates for PRs and issues

### PR DESCRIPTION
**Pull request by thephez**
### Issue being fixed or implemented  

This should help to make PRs/issues more consistent from all contributors by starting all new issues/PRs with the same templates as were added to various other platform repos. The PR template remains similar to the existing one.

### What was done  

Updated the PR template and added unique issue templates for both bug and feature request issues. The issue templates auto-assign the `bug` or `enhancement` label.

### How Has This Been Tested?
N/A

### Notes  
Coordinated initially in the mn-bootstrap repo. See dashevo/mn-bootstrap#47 and dashevo/mn-bootstrap#52.

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
-- [ ] My code follows the style guidelines of this project
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 - [ ] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added or updated relevant unit/integration/functional/e2e tests
 - [ ] I have made corresponding changes to the documentation
+
+**For repository code-owners and collaborators only**
+- [ ] I have assigned this pull request to a milestone
```
Comment (thephez) [inline .github/PULL_REQUEST_TEMPLATE.md:32]: Is there any plan to use milestones here like the other platform-related repos?

Comment (Alex-Werner) [reply .github/PULL_REQUEST_TEMPLATE.md:32]: Not that I'm aware of, but we did discuss about moving into that so we could highlight some wanted changes to the community so they are aware, so I guess we would go this route in the future. 
Right now, we do not.

Comment (thephez) [reply .github/PULL_REQUEST_TEMPLATE.md:32]: Okay, I will remove for now and we can add it back later if we start using milestones

**Comment by thephez** [comment]
@Alex-Werner Once #66 is merged, it should be possible to get this to pass tests. Or we could just merge as-is. Up to you :slightly_smiling_face:

**Comment by Alex-Werner** [comment]
I would say to merge it right away (so we won't have the old PR template on #66 and it's just github template), but let's still merge 66 first, as it will provide test of it working here I hope.
Reviewing right now :)

**Pull request merged**